source/backend/dal/Repositories/PropertyRepository.cs

```diff
@@ -195,7 +195,7 @@ public PimsProperty GetByPid(int pid, bool includeRetired = false)
 
             if(includeRetired)
             {
-                query = query.Where(r => r.IsRetired.HasValue && r.IsRetired.Value);
+                query = query.Where(r => !r.IsRetired.HasValue || (r.IsRetired.HasValue && r.IsRetired.Value));
```

Comment:
I think this would work, although it does look a tad odd. Maybe using the predicate builder and an Or statement would be more readable